src/constants.js

```diff
@@ -46,6 +46,8 @@ export const BLOB_UPDATE_OPERATIONS = {
 export const RECORD_IMPORT_STATE = {
 	CREATED: 'CREATED',
 	UPDATED: 'UPDATED',
+	ACTION_NEEDED: 'ACTION_NEEDED',
+	IN_QUEUE: 'IN_QUEUE',
```

Comment:
When Record-import importer sends record to Rest-api it sets it as IN_QUEUE.
Priority responce form Rest-api contains RECORD_IMPORT_STATE + payload.
e.g. That RECORD_IMPORT_STATE will inform if migrator to re handle that record

Reply:
The migrator uses the record import system to get the records into Melinda, but there should not any special functionality in the record import system for (It just has it's own transformer & importer as usual).

If the migrator needs special metadata, it should store them in it's own db/mq etc. Anyway, this is not relevant right now since migrator is not in development right now.